infrastructure/server/api-gateway.tf

```diff
@@ -66,10 +66,22 @@ resource "aws_api_gateway_deployment" "fam_api_gateway_deployment" {
   }
 
   rest_api_id = aws_api_gateway_rest_api.fam_api_gateway_rest_api.id
-  stage_name  = var.api_gateway_stage_name
+  stage_name  = var.api_gateway_stage_name  # Terraform warns about using "stage_name" in resource "aws_api_gateway_deployment".
+                                            # See https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/api_gateway_deployment
+                                            # If possible in the future, use "aws_api_gateway_stage" resource instead. But may face issue about:
```

Comment:
Did you create an issue in the backlog for this?

Reply:
Hi Basil, yes this is added into ticket #810 .